docs/getting-started-guides/azure.md

```diff
@@ -11,6 +11,8 @@ assignees:
 
 ## Overview
 
-The recommended approach for deploying a Kubernetes 1.4 cluster on Azure is the
-[`kubernetes-anywhere`](https://github.com/kubernetes/kubernetes-anywhere) project. You will want to take a look at the
-[Azure Getting Started Guide](https://github.com/kubernetes/kubernetes-anywhere/blob/master/phase1/azure/README.md).
\ No newline at end of file
+There are two approaches for deploying a Kubernetes 1.4 cluster on Azure. Those are: 
+
+* [Stackpoint.io](https://stackpoint.io/#/clusters/new?provider=azure) to build a cluster immediately.
```

Comment:
I think that this is better suited as a separate "Turn-Key Cloud Solution" since it is turn-key and it is multi-cloud.  So can we add a separate 'stackpoint.md' instead of adding stackpoint to each specific cloud provider?